Tasks/Maven/maventask.ts

```diff
@@ -203,6 +203,55 @@ function createMavenSQRunner(sqHostUrl, sqHostUsername, sqHostPassword, sqDbUrl?
     return mvnsq;
 }
 
+function processMavenOutput(data) {
+    data = data.toString();
+    var input = data;
+    var severity = 'NONE';
+    if(data.substring(0, 1) == '[') {
```

Comment:
Would "data.charAt(0)" be more appropriate for this check?  Also I think '===' is preferred over '=='.